sessions.go

```diff
@@ -134,11 +133,14 @@ func (s *Registry) Get(store Store, name string) (session *Session, err error) {
 		return nil, fmt.Errorf("sessions: invalid character in cookie name: %s", name)
 	}
 	if info, ok := s.sessions[name]; ok {
-		session, err = info.s, info.e
```

Comment:
Is this not now throwing away info.e?

Reply:
info.e isn't really useful in this case because it would've just been sending the same exact error back. Furthermore, the only place where the session error is saved (together with the session) is right after the session was created. However, if an error is found, it won't save the session (why should session data be saved for an invalid session?), so the only place where the error + session would've been saved will 100% of the time have a nil error which makes saving the error useless

Reply:
@elithrar ^